packages/f/fb2converter/package.yml

```diff
@@ -1,23 +1,27 @@
 name       : fb2converter
-version    : 1.74.1
-release    : 19
+version    : 1.74.4
+release    : 20
 source     :
-    - https://github.com/rupor-github/fb2converter/archive/refs/tags/v1.74.1.tar.gz : 370389df59b48d695b1865e3908b8613c47dc57d5a087c7f12bcb4c2aa49c0af
+    - git|https://github.com/rupor-github/fb2converter.git : v1.74.4
```

Comment:
Any reason you switched to git? We generally prefer tarballs unless they don't work for some reason

Reply:
That's my preference too. Unfortunately tarball didn't work, due to the build process using git cli for versioning purposes.

Reply:
That's fine then. In future, you can add a note to your git description to say: "Packaging note: Switched to git because upstream uses git to determine version"

If this project used a simpler build process for determining version I would ask you to use a tarball and set the version manually, but skimming the taskfile, that does not look straight forward